### PR TITLE
Missing Expense fields

### PR DESCRIPTION
**Pull request by bsonntag**
This adds some missing fields to Expenses and PeriodicExpenses:
- Supplier
- VAT
- Cost Center
- Payment Method

Also, `name` fields were renamed to `description`.

### CODE REVIEW

File: app/views/periodic_expenses/index.slim [hunk 1]
```diff
@@ -5,15 +5,36 @@
 - else
 
   table
```
Comment (zamith) [inline app/views/periodic_expenses/index.slim:7]: This looks like the exact same table as in the expenses. Is it? Will it always be?

File: app/views/periodic_expenses/new.slim [hunk 2]
```diff
@@ -1,16 +1,22 @@
 = simple_form_for @periodic_expense do |form|
-  = form.input :name
+  = form.input :start_date
 
-  = form.input :period, collection: PeriodicExpense.periods, label_method: :humanize
+  = form.input :end_date
 
-  = form.input :amount
+  = form.input :period, collection: PeriodicExpense.periods, label_method: :humanize
```
Comment (zamith) [inline app/views/periodic_expenses/new.slim]: This has that same issue of being coupled to the models, that we've talked about.

File: app/views/periodic_expenses/index.slim (hunk 1 above)
Comment (bsonntag) [inline app/views/periodic_expenses/index.slim:7]: It's almost the same, it has a few different fields. For instance, PeriodicExpense will always have a period while Expense will never have it.

Comment (zamith) [inline app/views/periodic_expenses/index.slim:7]: Ok.

File: app/views/periodic_expenses/new.slim (hunk 2 above)
Comment (bsonntag) [inline app/views/periodic_expenses/new.slim]: I moved these lines around and didn't notice it. Shame on me. :disappointed: 
Moved it to controller.

**Comment by zamith** [comment]
:+1:

**Pull request merged**